### PR TITLE
feat: APM transaction analysis dash

### PR DESCRIPTION
**Pull request by JimHagan**
# Summary

This quickstart adds a dashboard to make it easy to quickly analyze transaction performance for multiple transactions across hundreds of apps.  It makes use of our filtering capabilities to make drill down easy.

## Pre merge checklist

- [x ] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x ] Are all documentation links publicly accessible?
- [ x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x ] Did you check your descriptive content for spelling and grammar errors?
- [ x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ x] Does the PR contain a screenshot for each of your dashboards?
- [ x] Do your screenshots show data?
- [ x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### CODE REVIEW

File: quickstarts/apm-transaction-analysis/config.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  Analyze the performance of application transactions accross hundreds of applications using this filterable dashbaord.
+summary: |
+  A unified filterable view of transaction performance accross an account.
+level: New Relic
+authors:
+  - Jim Hagan
+keywords:
+  - APM
+  - Transaction
+installPlans:
```
Comment (brammerl) [inline quickstarts/apm-transaction-analysis/config.yaml:14]: We are no longer accepting `installPlans` as a field. Please remove this.

File: quickstarts/apm-transaction-analysis/config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+id: 870dc0f9-57d5-4532-a985-46f9aba019b5
+slug: apm-transaction-analysis
+title: APM Transaction Analysis
+description: |
+  Analyze the performance of application transactions accross hundreds of applications using this filterable dashbaord.
+summary: |
+  A unified filterable view of transaction performance accross an account.
```
Comment (brammerl) [inline quickstarts/apm-transaction-analysis/config.yaml:7]:
```suggestion
  A unified filterable view of transaction performance across an account.
```

File: quickstarts/apm-transaction-analysis/config.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+id: 870dc0f9-57d5-4532-a985-46f9aba019b5
+slug: apm-transaction-analysis
+title: APM Transaction Analysis
+description: |
+  Analyze the performance of application transactions accross hundreds of applications using this filterable dashbaord.
```
Comment (brammerl) [inline quickstarts/apm-transaction-analysis/config.yaml:5]:
```suggestion
  Analyze the performance of application transactions acc\ross hundreds of applications using this filterable dashboard.
```

[the remark this answers is not in the document]
Comment (JimHagan) [reply quickstarts/apm-transaction-analysis/config.yaml]: fixed

File: quickstarts/apm-transaction-analysis/config.yaml (hunk 2 above)
Comment (JimHagan) [reply quickstarts/apm-transaction-analysis/config.yaml]: fixed

File: quickstarts/apm-transaction-analysis/config.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+id: 870dc0f9-57d5-4532-a985-46f9aba019b5
```
Comment (aswanson-nr) [inline quickstarts/apm-transaction-analysis/config.yaml:1]: Let's remove this line, the system will auto generate an Id for new quickstarts!
```suggestion
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Looks like we have one thing to clean up before it's ready to merge!

**Comment by JimHagan** [comment]
@seemantk 

This note from the description tries to show how this dashboard offers a view that we can't/don't offer in the curated APM view:

```Normally in the curated APM view you will see one application at a time.  When you have hundreds of applications and wish to analyze the relative performance accross apps, this view excels.
  Using the power of NRQL we present a unified drill down of performance that is very useful when setting up alert conditions and service levels.  It is very good at helping explore performance outliers.```

File: quickstarts/apm-transaction-analysis/config.yaml (hunk 4 above)
Comment (JimHagan) [reply quickstarts/apm-transaction-analysis/config.yaml]: resolved.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**